### ISSUE TITLE
Link color on site is inaccessible

### ISSUE THREAD

**Issue by geoffrich** [sveltejs/site-kit#33]
The current color for links has a [color contrast ratio of 3.53](https://contrast-ratio.com/#%23ff3e00-on-white) against a white background, which could make it difficult for people with low vision to read. This fails the recommended [WCAG minimum contrast ratio](https://www.w3.org/TR/WCAG/#contrast-minimum) of 4.5.

I discovered this while working on other Axe accessibility issues I found on the Svelte docs (https://github.com/sveltejs/svelte/issues/5678).

Since this is [the primary color](https://github.com/sveltejs/site-kit/blob/01d83baee868053b01b560bc92ee34967616113f/base.css#L37) in the theme changing this could have wide-ranging effects. If we don't want to change this color, we could use a different color for links that passes the recommended contrast ratio threshold.

Example of link:
![link example](https://user-images.githubusercontent.com/4992896/100029820-f5683380-2da6-11eb-8de1-8ec5b7ab3760.png)

**Comment by geoffrich** [comment]
Also, the text color used for blockquotes on the docs [does not have have sufficient contrast](https://contrast-ratio.com/#hsl%28204%2C%20100%25%2C%2050%25%29-on-white).

Example:
![image](https://user-images.githubusercontent.com/4992896/100122885-cac1bd80-2e2e-11eb-8ca0-e1268cd63d5c.png)

**Comment by benmccann** [comment]
the homepage is currently getting 100 on accessibility from lighthouse

please let us know if any issues remain by filing at https://github.com/sveltejs/svelte.dev/issues
